src/box/iproto.cc

```diff
@@ -3107,3 +3079,22 @@ iproto_rmean_foreach(void *cb, void *cb_ctx)
 	}
 	return 0;
 }
+
+int
+iproto_rmean_foreach_thread(int thread_number, void *cb, void *cb_ctx)
+{
+	if (thread_number < 0 || thread_number >= iproto_threads_count)
+		return -1;
```

Comment:
We unable to add assert here, because `iproto_threads_count` is a private info for iproto.cc
User may want to enter invalid thread number and he should get `null`, not assertion.

Reply:
Let's make iproto_thread_count public. The code will be clearer then.

Reply:
ok fixed